### PR TITLE
feat: replace oclif with cmd-ts as oclif 

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: bin/sss [hunk 1]
```diff
@@ -1,8 +1,3 @@
 #!/usr/bin/env node
 
-require('@oclif/command')
-  .run()
-  .catch((e) => {
-    if (e.oclif != null) require('@oclif/errors/handle')(e);
-    else console.log(e);
-  });
+require('./sss.cjs');
```
Comment (blacha) [inline bin/sss:3]: the cjs file gets bundled before we deploy, so now there is just a single file.

File: package.json [hunk 2]
```diff
@@ -6,27 +6,27 @@
...
     "@types/node": "^16.0.1",
     "@types/ospec": "^4.0.2",
     "@types/pino": "^6.3.7",
     "@types/sinon": "^10.0.2",
     "@types/tar-stream": "^2.2.0",
+    "conventional-changelog-cli": "^2.1.1",
     "ospec": "^4.0.1",
-    "conventional-changelog-cli": "^2.1.1"
-  },
-  "dependencies": {
```
Comment (blacha) [inline package.json:22]: everything is now bundled so no runtime deps.

File: package.json [hunk 3]
```diff
@@ -6,27 +6,27 @@
...
+    "sinon": "^11.1.1",
     "@linzjs/s3fs": "^6.3.0",
-    "@oclif/command": "^1.8.0",
     "aws-sdk": "^2.888.0",
+    "cmd-ts": "^0.11.0",
```
Comment (Wentao-Kuang) [inline package.json:25]: This is quite new, are we giving up with `@rushstack/ts-command-line` as well? :smile:

File: src/commands/hash.ts [hunk 4]
```diff
@@ -35,20 +36,25 @@ export class HashManifest extends Command {
...
-    logger.info({ path: args.inputFile, count: manifest.files.size }, 'Hash:Done');
-  }
-}
+    logger.info({ path: args.manifest, count: manifest.files.size }, 'Hash:Done');
+  },
+});
 
 export async function hashFiles(toHash: ManifestFile[], manifest: ManifestLoader, Q = pLimit(5)): Promise<void> {
   let count = 0;
+  let startTime = performance.now();
```
Comment (paulfouquet) [inline src/commands/hash.ts:45]: Is that more precise that `Date.now()`?

File: src/commands/validate.ts [hunk 5]
```diff
@@ -107,5 +109,106 @@ export class ValidateManifest extends Command {
     } else {
       logger.info(stats, 'Validate:Done');
     }
-  }
-}
+  },
+});
+
+// export class ValidateManifest extends Command {
```
Comment (Wentao-Kuang) [inline src/commands/validate.ts:115]: Comment out code.

File: src/commands/hash.ts (hunk 4 above)
Comment (blacha) [reply src/commands/hash.ts:45]: Yes,

`Date.now()` gives ms percision as a int, `performance.now()` gives us precision as a floating point eg 1.324ms

File: package.json (hunk 3 above)
Comment (blacha) [reply package.json:25]: Yes, their is quite the conversation about cli tools in #cop-javascript :)

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**